alternator/expressions.cc

```diff
@@ -534,7 +541,7 @@ std::unordered_map<std::string_view, function_handler_type*> function_handlers {
                         format("{}: size() accepts 1 parameter, got {}", caller, f._parameters.size()));
             }
             rjson::value v = calculate_value(f._parameters[0], caller, previous_item);
-            return calculate_size(v);
+            return calculate_size(v, f._parameters[0].is_path());
```

Comment:
maybe we can have temporary reference and use it twice?

like `auto& arg = f._parameters[0]`